src/box/memtx_hash.c

```diff
@@ -369,9 +369,16 @@ memtx_hash_index_replace(struct index *base, struct tuple *old_tuple,
 				}
 			}
 			struct space *sp = space_cache_find(base->def->space_id);
-			if (sp != NULL)
-				diag_set(ClientError, errcode, base->def->name,
-					 space_name(sp));
+			if (sp != NULL) {
+			        if (errcode == ER_TUPLE_FOUND){
+			                diag_set(ClientError, errcode,  base->def->name,
```

Comment:
There's a mess of spaces and tabs. Please set up your editor to use tabs.

Reply:
fix